dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/DaoConfiguration.java

```diff
@@ -61,6 +63,11 @@ public MybatisPlusInterceptor paginationInterceptor(DbType dbType) {
         return interceptor;
     }
 
+    @Bean
+    public DatabaseIdProvider databaseIdProvider() {
+        return new FixedDatabaseIdProvider(daoPluginConfiguration.databaseId());
```

Comment:
Can we directly use daoPluginConfiguration.dbType().name() as databaseId?
I find the databaseId only exist in `TriggerRelationMapper.xml` if we modify the name to `POSTGRE_SQL` this should work.

Reply:
that's what I thought at first, use `DatabaseProductName` as databaseId. view commit https://github.com/apache/dolphinscheduler/pull/15153/commits/19d851011656c9fcc4dc5e07241fe98662340b8c

Reply:
If so, we can remove databaseId, and use the default `VendorDatabaseIdProvider` without add properties after change the datasourceId to PostgreSQL n mapper? .

Reply:
yup

Reply:
Please remove databaseId in DaoPlugin, and use this way.

Reply:
Can I close this PR and recreate it? Too many commits are not clean enough

Reply:
You can rebase the commit, and I will squash the commit when merged. So this is not a problem, don't worry.